kernel/pit_clock/src/lib.rs

```diff
@@ -21,7 +21,7 @@ use pit_clock_basic::*;
 
 /// The PIT Channel 0 is connected directly to IRQ 0.
 /// Because we perform the typical PIC remapping, the remapped IRQ vector number is 0x20.
-const PIT_CHANNEL_0_IRQ: u8 = interrupts::IRQ_BASE_OFFSET + 0x0;
+const PIT_CHANNEL_0_IRQ: u8 = interrupts::IRQ_BASE_OFFSET;
```

Comment:
this is another case where the logic is encoding some information, so let's undo this change and allow this identity op lint.